Solutions/VaronisSaaS/Data Connectors/azuredeploy.json

```diff
@@ -0,0 +1,257 @@
+      "defaultValue": "[format('VaronisSaaS-{0}', uniqueString(resourceGroup().id))]",
+      "metadata": {
+        "description": "The name of the Azure Function app."
+      }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS",
+        "Standard_RAGRS"
+      ],
+      "metadata": {
+        "description": "Storage Account type"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "appInsightsLocation": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for Application Insights"
+      }
+    },
+    "functionWorkerRuntime": {
+      "type": "string",
+      "defaultValue": "dotnet",
+      "allowedValues": [
+        "dotnet"
+      ],
+      "metadata": {
+        "description": "The language worker runtime to load in the function app."
+      }
+    },
+    "functionPlanOS": {
+      "type": "string",
+      "defaultValue": "Linux",
+      "allowedValues": [
+        "Windows",
+        "Linux"
+      ],
+      "metadata": {
+        "description": "Specifies the OS used for the Azure Function hosting plan."
+      }
+    },
+    "functionAppPlanSku": {
+      "type": "string",
+      "defaultValue": "Y1",
+      "allowedValues": [
+        "Y1",
+        "EP1",
+        "EP2",
+        "EP3"
+      ],
+      "metadata": {
+        "description": "Specifies the Azure Function hosting plan SKU."
+      }
+    },
+    "packageUri": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/VaronisSaaS/Data%20Connectors/Varonis.Sentinel.Functions.zip",
+      "metadata": {
+        "description": "The zip content url."
+      }
+    },
+    "linuxFxVersion": {
+      "type": "string",
+      "defaultValue": "DOTNET|6.0",
+      "metadata": {
+        "description": "Only required for Linux app to represent runtime stack in the format of 'runtime|runtimeVersion'. For example: 'python|3.9'"
+      }
+    },
+    "logAnalyticsWorkspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Log Analytics workspace used by Microsoft Sentinel."
+      }
+    },
+    "datAlertHostName": {
+      "type": "string",
+      "metadata": {
+        "description": "Varonis DatAlert host name."
+      }
+    },
+    "datAlertApiKey": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Varonis DatAlert API key."
+      }
+    }
+  },
+  "variables": {
+    "hostingPlanName": "[parameters('functionAppName')]",
+    "applicationInsightsName": "[parameters('functionAppName')]",
+    "storageAccountName": "[format('{0}azfunctions', uniqueString(resourceGroup().id))]",
```

Comment:
Hello @vkorenkov-varonis, replace azfunction with SA or limited character word. 
including azfunction is crossing the storagename limit. 
also can you share working images and output of function app.

Reply:
Renaming has been done. Output you can find here:
![image](https://github.com/Azure/Azure-Sentinel/assets/99330808/933cddda-0e25-41fb-8030-d37e70f1942f)

Not sure what do you mean by working images?
Workbook images can be found here:
![image](https://github.com/Azure/Azure-Sentinel/assets/99330808/66df257e-67dc-459e-aa86-0f2012e8949a)